gnu.elf/plugin.xml

```diff
@@ -2124,6 +2234,55 @@
             name="%toolchain.hseb"
             superClass="arc.gnu.elf.toolchain.arceb">
       </toolChain>
+      <toolChain
+            id="arc.gnu.elf.toolchain.hs6x"
+            name="%toolchain.hs6x"
+            superClass="arc.gnu.elf.toolchain.arc">
+         <tool
+               id="arc.gnu.elf.tool.c.compiler.hs6x"
```

Comment:
This needs a `command="arc64-elf-gcc"` as well, otherwise it will call arc-elf32-gcc instead/